server/edusync/src/main/java/com/codestates/edusync/searchtag/repository/SearchTagRepository.java

```diff
@@ -8,4 +8,5 @@
 public interface SearchTagRepository extends JpaRepository<SearchTag, Long> {
 
     List<SearchTag> findAllByTagKey(String key);
+    List<SearchTag> findByStudygroupId(Long studygroupId);
```

Comment:
```suggestion
    List<SearchTag> findAllByStudygroupId(Long studygroupId);
```
여러개를 받기 위해서는 findAllBy 로 시작해야할겁니다

Reply:
그렇네요!!